ospool-pilot/itb/pilot/where-am-i

```diff
@@ -101,20 +104,20 @@ else
   gconfig_add "PublicIPV6" "Unknown"
 fi
 
-ping -c 1 google.com
-ping_error_code=$?
+python3 ospool-pilot/itb/lib/tcpping.py -n 1 google.com
```

Comment:
Another issue here, it is possible (even if unlikely) that `python3` may not be available everywhere. So you might want to also do an existence check for `python3` and fall back to `python` if it doesn't. You could use the same trick with environment variables:

```shell
if <python3 is available>; then
  PYTHON="python3"
else
  PYTHON="python"
fi

...

TCPPING="${PYTHON} ${group_dir}/tcpping.py"
```